hack/install.sh

```diff
@@ -141,7 +141,7 @@ do_install() {
 	esac
 
 	if command_exists docker; then
-		version="$(docker -v | awk -F '[ ,]+' '{ print $3 }')"
+		version="$(docker -v | cut -d ' ' -f3 | cut -d ',' -f1)"
```

Comment:
Replaced `awk` with `cut` for better compatibility (`awk` is not installed by default on Photon OS)

cc: @tiborvass